src/client/Microsoft.Identity.Client/Internal/ClientCredential/CertificateAndClaimsClientCredential.cs

```diff
@@ -20,29 +17,26 @@ internal class CertificateAndClaimsClientCredential : IClientCredential
     {
         private readonly IDictionary<string, string> _claimsToSign;
         private readonly bool _appendDefaultClaims;
-
+        private readonly string _base64EncodedThumbprint; // x5t
         public X509Certificate2 Certificate { get; }
 
         public AssertionType AssertionType => AssertionType.CertificateWithoutSni;
 
-        public CertificateAndClaimsClientCredential(
-            X509Certificate2 certificate,
-            IDictionary<string, string> claimsToSign, 
-            bool appendDefaultClaims)
+        public CertificateAndClaimsClientCredential(X509Certificate2 certificate, IDictionary<string, string> claimsToSign, bool appendDefaultClaims)
```

Comment:
Why remove the stacking of these parameters? Do we have any general guidance on this?
Generally, if the parameters line has a bunch of characters, I always stack them.